experimental/responsive-design/src/lib/components/information-window.js

```diff
@@ -0,0 +1,116 @@
+import { LitElement, html, adoptStyles } from "lit";
+import { restaurants } from "../../data/restaurants.js";
+import "./restaurant-card.js";
+
+import chatWindowStyles from "../chat-window.constructable.js";
+class InformationWindow extends LitElement {
+    static properties = {
+        restaurants: { type: Array },
+        _isChatExpanded: { type: Boolean },
+        _currentIndex: { type: Number },
+        chatWindow: { type: Object },
+    };
+
+    constructor() {
+        super();
+        this.restaurants = restaurants;
+        this._isChatExpanded = true;
+        this._currentIndex = 0;
+        this._resizeObserver = new ResizeObserver((entries) => {
+            for (const entry of entries) {
+                if (entry.contentBoxSize && entry.contentBoxSize[0])
+                    this._isChatExpanded = entry.contentBoxSize[0].blockSize > 350;
+                else
+                    this._isChatExpanded = entry.contentRect.height > 350;
+                this._currentIndex = 0;
+                this.updateCarousel();
+            }
+        });
+    }
+
+    connectedCallback() {
+        super.connectedCallback();
+        adoptStyles(this.shadowRoot, [chatWindowStyles]);
```

Comment:
Don't we want / need to re-install resize observer here?
Same as other classes that use resize observer and disconnect in `disconnectedCallback`.